kindlecomicconverter/comic2ebook.py

```diff
@@ -1154,9 +1163,17 @@ def makeBook(source, qtgui=None):
     checkPre(source)
     print("Preparing source images...")
     path = getWorkFolder(source)
+    pathOebpsImages = os.path.join(path, "OEBPS", "Images")
     print("Checking images...")
-    getComicInfo(os.path.join(path, "OEBPS", "Images"), source)
-    detectCorruption(os.path.join(path, "OEBPS", "Images"), source)
+    getComicInfo(pathOebpsImages, source)
+    listFiles, listCbzFiles, countCbzFiles = listFilesS(pathOebpsImages), listCbzFilesS(pathOebpsImages), countCbzFilesS(pathOebpsImages)
+    if countCbzFiles:
```

Comment:
This variable is unneccessary, just use `len(listCbzFiles)`

Reply:
okay, I will fix it